### PR TITLE
Shoop 2.0.0

### PR DESCRIPTION
**Pull request by akx**

### CODE REVIEW

File: ChangeLog.rst [hunk 1]
```diff
@@ -9,6 +9,94 @@ Unreleased
   releasing next version, the "Unreleased" header will be replaced with
   appropriate version header and this help text will be removed.
 
+Version 2.0.0
+-------------
```
Comment (suutari-ai) [inline ChangeLog.rst]: Don't put the 2.0.0 header just yet. See 81fbe29feb037fb8dc32c252fac097c8fbd17ccd and e3dd81627de41648de90833c3555c3a65ac2b991.

File: ChangeLog.rst [hunk 2]
```diff
@@ -9,6 +9,94 @@ Unreleased
...
+- Time interval attribute now renders as `DecimalField` (45e774f)
+- Notify: Add better error handling for "step edit"-popup (df76c12, 2366ff1)
+- Better variation error handling (a877781)
+- Fix issue where visibility errors caused an error (acda42f)
+- Update admin category view (a404fa0)
+- Admin datetimepicker (f1681e5)
+
+Core
+~~~~
+
```
Comment (suutari-ai) [inline ChangeLog.rst]: Maybe core should be first?

File: ChangeLog.rst [hunk 3]
```diff
@@ -9,6 +9,94 @@ Unreleased
...
+Classic Gray Theme
+~~~~~~~~~~~~~~~~~~
+
+- A new slick theme built on the Xtheme system (d168678, 1636057, a1c1260,
+  ed8712e, 4da7ef0, 4ffb38a, cd76bd2, af033ab, 466300a, 37be170, 2d2f59f,
+  bca18c7, 385fe82, 7fe20dd, 4d39fbd, 21a2a4d, 4efd82a, f430bd0, 7417150,
+  05703dc, 42d6fdc, 65ab2cd, 7058a25, aec8899, c9040ad, 87e9792, 169f002,
+  dcd9219, 6cb9340, 863ecb9, 2dd696d, 5e00137, 4eecb40, 0ddbb8b, 06fad94,
+  15818c8, fc49732, 379806e, 9d2986f, bd77430, 18de0b9, 2dd853a, ee73697,
+  e81ab3b)
```
Comment (suutari-ai) [inline ChangeLog.rst]: Hmm.. so many refs... doesn't look pretty.

File: ChangeLog.rst [hunk 4]
```diff
@@ -9,6 +9,94 @@ Unreleased
...
+- Fix checking of duplicate settings (685d393)
+- Add new fields to shop core (d56dd24)
+- Add `DiscountPricingModule` (SHOOP-1289) (f9f3789)
+- Fix PriceInfo usage with non-one quantities (SHOOP-1462) (1bb1c02)
+- Fix shoop.core.migrations.0006 (b605d04)
+- Tax clean-up and refactoring (SHOOP-975) (2cb5a50, 182daff)
+- Prevent `Shop` being deleted when image was deleted (c0edb08)
+
+Frontend
+~~~~~~~~
```
Comment (suutari-ai) [inline ChangeLog.rst]: Are you referring to "shoop.front"? I thought we've named it "Shoop Front"... maybe this subtitle should be just "Front"?

**Comment by suutari-ai** [comment]
The dividing the changes to Core/Admin/etc. groups is great! Good job!

File: ChangeLog.rst (hunk 1 above)
Comment (akx) [inline ChangeLog.rst]: This commit is equivalent to 81fbe29feb037fb8dc32c252fac097c8fbd17ccd though?

File: ChangeLog.rst (hunk 2 above)
Comment (akx) [inline ChangeLog.rst]: They are alphabetically sorted.

File: ChangeLog.rst (hunk 1 above)
Comment (suutari-ai) [inline ChangeLog.rst]: Then add the released on line too and remove the unreleased section:
```
Released on 2015-10-05 16:30 +0300.
```

Comment (akx) [inline ChangeLog.rst]: I don't see why bother "churning" the Unreleased section?

Comment (suutari-ai) [inline ChangeLog.rst]: It's a cruft that just helps out writing the changelog while developing, but it shouldn't be there to distract downloaders of the release. (It's not intended for them.)

File: setup.py [hunk 5]
```diff
@@ -38,7 +38,7 @@
 #      - Add ".post0.dev" suffix to VERSION variable here
 
 NAME = 'shoop'
-VERSION = '1.2.0.post0.dev'
+VERSION = '2.0'
```
Comment (suutari-ai) [inline setup.py]: Should be 2.0.0

**Pull request merged**